src/FunctionalArrayCollection.js

```diff
@@ -0,0 +1,69 @@
+import FunctionalCollection from './FunctionalCollection'
+
+/**
+ * Functional collection that works with arrays under the hood
+ */
+export class FunctionalArrayCollection extends FunctionalCollection {
+    /**
+     * Retrieves items of the collection.
+     *
+     * @return {Array} The items, by item ID.
+     */
+    getItems () {
+        return this._getItems().slice()
+    }
+
+    /**
+     * Check that given item in the collection.
+     *
+     * @param item
+     * @returns {boolean}
+     */
+    hasItem (item) {
+        let foundItemIndex = null
+        let items = this.getItems()
```

Comment:
This makes a copy of the entire set of items every time you check for an item. Tsk tsk tsk.

Reply:
I'll fix 👍